src/ec/packages/ec-component-hero-banner/hero-banner.story.js

```diff
@@ -15,105 +16,97 @@ import heroBanner from './ecl-hero-banner.html.twig';
 import notes from './README.md';
 
 const PrepareBanner = data => {
+  getExtraKnobs(data);
+  if (data.link.link.label) {
+    data.link.link.label = text(
+      'Link label',
+      data.link.link.label,
+      buttonLabels.required
+    );
+  }
+
+  if (data.title) {
+    data.title = text('Title', data.title, buttonLabels.required);
+  }
+
+  if (data.title) {
+    data.title = text('Title', data.title, buttonLabels.required);
+  }
+
+  if (data.description) {
+    data.description = text(
+      'Description',
+      data.description,
+      buttonLabels.required
+    );
+  }
+
+  if (data.centered) {
+    data.centered = boolean('Centered', data.centered, buttonLabels.optianal);
```

Comment:
I believe this should be buttonLabels.optional in your intention :)
But at the same time, although is not entirely proper, i would place this a "state".
At the moment, because of this typo, you have the "other" tab in the knobs, this will disappear once you apply the right label.

Reply:
Ok, you fixed the typo but the "centered" parameter has not been moved to the "states" group.
The reason why i think is the right thing to do is that this option should be available for all the variants, but the "align-left" one (it's also like this in ECL). 
So, my recommendation would be to set the "centered" parameter for all the variants in the "states" group excluding the "align-left" one.